modules/salt/pages/yomi.adoc

```diff
@@ -282,7 +282,7 @@ This behavior is set to be changed in future versions of Yomi.
 
 * User 1:
 ** Username: root
-** Password Hash: $1$wYJUgpM5$RXMMeASDc035eX.NbYWFl0
+** Password Hash: $1$wYJUgpM5$RXMMeASDc035eXNbYWFl0
```

Comment:
I know technically this wasn't an "error" but it's easier to just remove the period so the linter doesn't complain, I think.